### PR TITLE
Miscellaneous minor formatting and wording improvements

### PR DESCRIPTION
**Pull request by sschuberth**
Please have a look at the individual commit messages for the details.

### CODE REVIEW

File: etc/conf/base.py [hunk 1]
```diff
@@ -1,3 +1,7 @@
+"""
+This base config script gets automatically executed for all platforms
+via configure if no sub-configuration is specified.
```
Comment (pombredanne) [inline etc/conf/base.py]: Actually I think the behavior is that it gets automatically executed in all cases.

Comment (sschuberth) [inline etc/conf/base.py]: Done.

File: scancode.bat [hunk 2]
```diff
@@ -1,34 +1,31 @@
 @echo OFF
-@rem  Copyright (c) 2015 nexB Inc. http://www.nexb.com/ - All rights reserved.
-@rem  
-
+@rem Copyright (c) 2015 nexB Inc. http://www.nexb.com/ - All rights reserved.
 
 @rem  A minimal shell wrapper to the CLI entry point
 
 set SCANCODE_ROOT_DIR=%~dp0
-set SCANCODE_CMD_LINE_ARGS=
```
Comment (pombredanne) [inline scancode.bat]: I think that's part of the joy of Windows BAT scripting ;)

Comment (sschuberth) [inline scancode.bat]: Very good point, thanks for noticing. Fixed.

**Comment by pombredanne** [comment]
Thanks! LGTM, merging!

**Pull request merged**